src/app/components/member/member-profile/member-profile.component.html

```diff
@@ -1,60 +1,86 @@
+          <div class="row">
+            <div class="col-12">
+              <ul class="nav nav-tabs mb-4" id="myTab" role="tablist">
+                <li class="nav-item">
+                  <a class="nav-link active" id="basicInfo-tab" data-toggle="tab" href="#basicInfo" role="tab"
+                    aria-controls="basicInfo" aria-selected="true">Basic Info</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" id="connectedServices-tab" data-toggle="tab" href="#connectedServices" role="tab"
+                    aria-controls="connectedServices" aria-selected="false">Connected Services</a>
+                </li>
+              </ul>
+              <div class="tab-content ml-1" id="myTabContent">
+                <div class="tab-pane fade show active" id="basicInfo" role="tabpanel" aria-labelledby="basicInfo-tab">
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">ID</label>
+                    </div>
+                    <div class="col-md-8 col-6">{{member.id}} </div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">First Name</label>
+                    </div>
+                    <div class="col-md-8 col-6">{{member.firstName}} </div>
+                  </div>
+                  <hr />
 
-<div class="row">
-<div class="col-sm-3 col-md-2 col-5">
-<label style="font-weight:bold;">Last Name</label>
-  </div>
-    <div class="col-md-8 col-6"> {{lastname}}</div>
-       </div>
-    <hr />
-    <div class="row">
-     <div class="col-sm-3 col-md-2 col-5">
-<label style="font-weight:bold;">Email</label>
-     </div>
- <div class="col-md-8 col-6"> {{email}} </div>
- </div>
-                                   
- </div>
- <div class="tab-pane fade" id="connectedServices" role="tabpanel" aria-labelledby="ConnectedServices-tab">
-   Facebook, Google, Twitter Account that are connected to this account
-  </div>
-                        
-</div>
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">Last Name</label>
+                    </div>
+                    <div class="col-md-8 col-6"> {{member.lastName}}</div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">Gender</label>
+                    </div>
+                    <div class="col-md-8 col-6"> {{member.gender}} </div>
+                  </div>
+                  <hr />
+                  <div class="row">
+                    <div class="col-sm-3 col-md-2 col-5">
+                      <label style="font-weight:bold;">DateOfBirth</label>
```

Comment:
The label needs to be readable as much as possible. DateOfBirth is good for variable but not for label - make it Date of Birth.